### ISSUE TITLE
Serializing Successful - nullpointer in Result.getErrorMessage

### ISSUE THREAD

**Issue by FloKaemmerer** [damianszczepanik/cucumber-reporting#796]
Im trying to serialize the List of Features returned by reportParser.parseJsonFiles().
sadly I endup getting a NPE.

> "Could not write JSON: (was java.lang.NullPointerException); nested exception is com.fasterxml.jackson.databind.JsonMappingException: (was java.lang.NullPointerException) (through reference chain: java.util.ArrayList[0]->net.masterthought.cucumber.json.Feature[\"elements\"]->net.masterthought.cucumber.json.Element[0]->net.masterthought.cucumber.json.Element[\"steps\"]->net.masterthought.cucumber.json.Step[0]->net.masterthought.cucumber.json.Step[\"result\"]->net.masterthought.cucumber.json.Result[\"errorMessageTitle\"])"

To me it looks like the errorMessage is not set in the Result object when you have a Successfull Feature.
Therefore calling `getErrorMessageTitle()` results in an NPE:
`String[] title = errorMessage.split("[\\p{Space}]+");`

```
public class Result implements Durationable {
    @JsonProperty("error_message")
    private final String errorMessage = null;

public final String getErrorMessageTitle() {
        String[] title = errorMessage.split("[\\p{Space}]+"); //NPE when called without setting the errorMessage
        if (title.length > 0) {
            return title[0];
        }
        return StringUtils.EMPTY;
    }
}
```

**Comment by FloKaemmerer** [comment]
see the attached file.

Setting the value of `errorMessage` to an empty String instead of null solved the Problem of parsing the result.
But I run into the next NPE:
```"Could not write JSON: (was java.lang.NullPointerException); nested exception is com.fasterxml.jackson.databind.JsonMappingException: (was java.lang.NullPointerException) (through reference chain: java.util.ArrayList[0]->net.masterthought.cucumber.json.Feature[\"passedFeatures\"])"```

[report.json.txt](https://github.com/damianszczepanik/cucumber-reporting/files/2657286/report.json.txt)

**Comment by damianszczepanik** [comment]
Have you read this documentation https://github.com/damianszczepanik/cucumber-reporting ? Please

**Comment by FloKaemmerer** [comment]
Yes I've read the documentation and yes I know that the project is mainly to generate html reports.
But nevertheless I would expect not to run in NPEs or "Not Implemented yet Exceptions" when working with the Objects returned by the lib.

I would be realy greatfull if you could take a look at the NPEs.

**Comment by damianszczepanik** [comment]
NPE does not happen if you follow instructions how to generate the report

**Comment by damianszczepanik** [comment]
I was able to generate report and evaluate returned object against null
